Tests/Functional/DSL/Aggregation/AvgAggregationTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Functional\DSL\Aggregation;
+
+use ONGR\ElasticsearchBundle\DSL\Aggregation\AvgAggregation;
+use ONGR\ElasticsearchBundle\ORM\Repository;
+use ONGR\ElasticsearchBundle\Test\AbstractElasticsearchTestCase;
+
+class AvgAggregationTest extends AbstractElasticsearchTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function getDataArray()
+    {
+        return [
+            'default' => [
+                'product' => [
+                    [
+                        '_id' => 1,
+                        'title' => 'foo',
+                        'price' => 10.45,
+                    ],
+                    [
+                        '_id' => 2,
+                        'title' => 'bar',
+                        'price' => 32,
+                    ],
+                    [
+                        '_id' => 3,
+                        'title' => 'pizza',
+                        'price' => 15.1,
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * Test for avg aggregation.
+     */
+    public function testMinAggregation()
+    {
+        /** @var Repository $repo */
+        $repo = $this->getManager()->getRepository('AcmeTestBundle:Product');
+
+        $aggregation = new AvgAggregation('test_agg');
+        $aggregation->setField('price');
+
+        $search = $repo->createSearch()->addAggregation($aggregation);
+        $results = $repo->execute($search, Repository::RESULTS_RAW);
+
+        $expectedResult = [
+            'agg_test_agg' => [
+                'value' => 19.18,
+            ],
+        ];
+
+        $this->assertArrayHasKey('aggregations', $results);
```

Comment:
When test case has more than one assertion try to provide a message for each assert